imports/camera/upload-photos.ts

```diff
@@ -0,0 +1,71 @@
+import { PACKAGE_NAME } from './initialize-package';
+
+export default async function uploadPhotos(deep, deviceLinkId, photos) {
+    const cameraLinkId = await deep.id(deviceLinkId, "Camera");
+    const containTypeLinkId = await deep.id("@deep-foundation/core", "Contain");
+    const photoTypeLinkId = await deep.id(PACKAGE_NAME, "Photo");
+    const base64TypeLinkId = await deep.id(PACKAGE_NAME, "Base64");
+    const webPathTypeLinkId = await deep.id(PACKAGE_NAME, "WebPath");
+    const exifTypeLinkId = await deep.id(PACKAGE_NAME, "Exif");
+    const formatTypeLinkId = await deep.id(PACKAGE_NAME, "Format");
+    const timestampTypeLinkId = await deep.id(PACKAGE_NAME, "TimeStamp");
+
+    const { data: [{ id: photoLinkId }] } = await deep.insert(photos.map((photo) => ({
+        type_id: photoTypeLinkId,
+        in: {
+            data: [{
+                type_id: containTypeLinkId,
+                from_id: cameraLinkId,
+            }]
+        },
+        out: {
+            data: [
+                {
+                    type_id: containTypeLinkId,
+                    to: {
+                        data: {
+                            type_id: base64TypeLinkId,
+                            string: { data: { value: photo.base64String } },
+                        }
+                    }
+                },
+                {
+                    type_id: containTypeLinkId,
+                    to: {
+                        data: {
+                            type_id: webPathTypeLinkId,
+                            string: { data: { value: photo.webPath ? photo.webPath : "none" } },
+                        }
+                    }
+                },
```

Comment:
```suggestion
                photo.webPath && {
                    type_id: containTypeLinkId,
                    to: {
                        data: {
                            type_id: webPathTypeLinkId,
                            string: { data: { value: photo.webPath } },
                        }
                    }
                },
```

Do not create links if value is null/undefined.

_This comment applies to all the places working like these lines of code_